CDCgov/prime-reportstream
# Fixed the issue where it could not handle Piped targetTypes

This PR fixed the issue where we do not know how to handle piped targetTypes in the FhirBundleUtils. 

Test Steps:
1. Make sure the tests pass.
2. Make sure that when you run a test through the API that it no longer has errors in the logs pertaining to piped targetTypes
3. Ran a message before and ran a message after changes and made sure that the contents were identical. 

## Changes
Added handling of piped targetTypes

## Checklist

### Testing

### Process

## Linked Issues
- #issue #12708

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
